modules/services/event-bridge/organizational.tf

```diff
@@ -8,14 +8,91 @@ data "aws_organizations_organization" "org" {
   count = var.is_organizational ? 1 : 0
 }
 
+data "aws_region" "current" {}
+
 locals {
   organizational_unit_ids = var.is_organizational && length(var.organization_units) == 0 ? [for root in data.aws_organizations_organization.org[0].roots : root.id] : toset(var.organization_units)
 }
 
+# permission policy for stackset admin role
+resource "aws_iam_policy" "admin_role_policy" {
+  name   = "stackset-admin-role-policy"
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "sts:AssumeRole"
+            ],
+            "Resource": [
+                "arn:*:iam::*:role/AWSCloudFormationStackSetExecutionRole"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+EOF
+}
+
+# policy attachment for stackset admin role
+resource "aws_iam_role_policy_attachment" "admin_role_attachment" {
+  policy_arn = aws_iam_policy.admin_role_policy.arn
```

Comment:
Do we need a `count = var.is_organizational ? 1 : 0` here?

Reply:
added count here.